consensus/consensus.md

```diff
@@ -71,22 +71,28 @@ account to smart contracts.
 
 ### Transaction signature
 
-We sign serialized transaction prefixed with the id of the network.
-See [serialization](../serializations.md#binary-serialization) definition for details.
+We sign serialized transaction (or the hash of the serialized transaction -
+from Lima hard-fork) prefixed with the id of the network. See
+[serialization](../serializations.md#binary-serialization) definition for
+details.
 
 ```
 NetworkId             :: binary()
 SerializedObject      :: binary()
 Signature             :: binary()
 
 Signature = sign(NetworkId + SerializedObject)
+or Signature = sign(NetworkId + Blake2b(SerializedObject))
 ```
 
 Prefix defaults to ``ae_mainnet`` (``binary()``) and it is configurable via node config.
 The prefix is not part of a serialized transaction. We add it only for signature.
 Consider changing the id in case of forking the blockchain network,
 in order to lower danger of replay attacks.
 
+**Note:** Post Lima hard-fork it is allowed to sign either the full serialized object, or
```

Comment:
The definition of "Lima" is introduced in https://github.com/aeternity/protocol/pull/391

It would be great if you could have a look to that one too.